### PR TITLE
Test Framework changes for parquet filter pushdown project.

### PR DESCRIPTION
**Pull request by rhou1**
Hi Chun,

Can you review these changes?

Thanks.

--Robert

### CODE REVIEW

File: framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java [hunk 1]
```diff
@@ -500,19 +502,23 @@ public TestStatus verifyTextPlan(String expectedOutput,
     throw new VerificationException(sb.toString());
   }
 
-  private static boolean matchesAll(String actual, String expected) {
+  private static boolean matchesAll(String actual, String expected,
+                                    boolean noOrder) {
```
Comment (cchang738) [inline framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java]: Let's change the boolean name to checkOrder. We don't want to use double negative (!noOrder). Change it to 'if (checkOrder) {}

File: framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java [hunk 2]
```diff
@@ -500,19 +502,23 @@ public TestStatus verifyTextPlan(String expectedOutput,
...
       Matcher matcher = Pattern.compile(string).matcher(actual);
-      if (matcher.find()) {
+      if (!matcher.find()) {
+        return false;
+      }
+      if (!noOrder) {
         String matched = matcher.group();
         i = actual.indexOf(matched);
         actual = actual.substring(i + matched.length()).trim();
       } else {
```
Comment (cchang738) [inline framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java]: remove the else block.

**Comment by rhou1** [comment]
Hi Chun,

I have updated the code.  Please review them.

Thanks.

--Robert

**Comment by rhou1** [comment]
Hi Chun,

I have updated the code. Please review them.

Thanks.

--Robert

**Comment by cchang738** [comment]
LGTM. +1

Please make sure you run pre-commit test with the code change. Thanks for the improvement.

**Pull request merged**